Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -195,7 +191,16 @@ void PlayerBase::Load()
 					}
 					else if (ini.is_value("system"))
 					{
-						system = ini.get_value_int(0);
+						string sysNickname = ini.get_value_string(0);
```

Comment:
??? so this field now has two valid types, meaning either a string nickname or its hash? why would we need to store system as anything other than a hash?

Reply:
For convenience of admins reading/writing to those files. If human-readability wasn't a factor, we'd use single character variable names to save on space and performance :]